tests/data/xref.json

```diff
@@ -0,0 +1,1739 @@
+{
+  "data": [
```

Comment:
We should probably just include what we are testing.

Reply:
Now that it's in commit history, I'll do it :)